pages/agent/v3/cli_step.md

```diff
@@ -4,13 +4,31 @@ The Buildkite agent's `step` command provides the ability to retrieve and update
 
 ## Updating a step
 
-Use this command in your build scripts to update an attribute of a step.
+Use this command in your build scripts to update an attribute of a step. The following attributes can be updated:
+
+* `label`
```

Comment:
A list containing only a single item looks rather odd. Unless we're intending to add more items to this list in the very near future, I'd recommend simplifying this to:

```suggestion
Use this command in your build scripts to update the `label` attribute of a step.
```